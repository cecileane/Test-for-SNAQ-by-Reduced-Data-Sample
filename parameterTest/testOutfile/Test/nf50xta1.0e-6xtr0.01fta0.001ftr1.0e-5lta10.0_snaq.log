optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=0.001,
                       xtolAbs=1.0e-6, xtolRel=0.01.
 max number of failed proposals = 50, liktolAbs = 10.0.
rootname for files: nf50xta1.0e-6xtr0.01fta0.001ftr1.0e-5lta10.0_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 70342
seed: 70342 for run 1
Sat Oct  1 17:18:22 2016

 BEGIN SNaQ for run 1, seed 70342 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 63 (failures: 50) with -loglik=128.22643 and ht_min=[0.30771,1.73904,0.66107,8.87897,0.4599]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 17		 17		 1	 |	 0		 0		 --
mvtarget	 20		 20		 1	 |	 0		 0		 --
chdir		 7		 7		 1	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 18		 14		 0	 |	 2		 2		 --
Total		 63		 59		 4	 |	 2		 2		 2
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 1, -loglik of best 128.22642796573987
(2,((3,(5,(6)#H7:::0.6922885428890575):1.739042770218895):0.4598966544335186,4):0.6610694049213363,(1,#H7:::0.30771145711094255):8.878969856269661);
---------------------
seed: 12846 for run 2
Sat Oct  1 17:18:27 2016

 BEGIN SNaQ for run 2, seed 12846 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 107 (failures: 50) with -loglik=128.22643 and ht_min=[0.30771,0.66107,1.73902,0.4599,8.87903]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 4		 4		 1	 |	 NA 		 NA 		 NA 
mvorigin	 29		 29		 2	 |	 0		 0		 --
mvtarget	 21		 21		 0	 |	 1		 1		 --
chdir		 11		 11		 1	 |	 1		 1		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 42		 29		 1	 |	 6		 6		 --
Total		 107		 94		 5	 |	 8		 8		 6
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 0.8

 FINISHED SNaQ for run 2, -loglik of best 128.22642796014017
((2,(4,(3,((6)#H7:::0.6922899546046088,5):1.739019537243421):0.45989628065524363):0.6610706797906918):8.879032997766629,1,#H7:::0.3077100453953911);
---------------------

Sat Oct  1 17:18:34 2016
MaxNet is ((2,(4,(3,((6)#H7:::0.6922899546046088,5):1.739019537243421):0.45989628065524363):0.6610706797906918):8.879032997766629,1,#H7:::0.3077100453953911); 
with -loglik 128.22642796014017
