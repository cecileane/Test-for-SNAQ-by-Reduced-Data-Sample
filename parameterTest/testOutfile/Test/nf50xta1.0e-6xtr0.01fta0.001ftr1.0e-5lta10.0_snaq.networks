((2,(4,(3,((6)#H7:::0.6922899546046088,5):1.739019537243421):0.45989628065524363):0.6610706797906918):8.879032997766629,1,#H7:::0.3077100453953911);, with -loglik 128.22642796014017 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(4,(3,((6,#H7:::0.28617947103978314):5.214810762499793,5):0.5143245743896697):0.47255870803837563,((1)#H7:::0.7138205289602169,2):1.5428048652308293);, with -loglik 126.8407629781011
((2,#H7:::0.38359521445537736):8.228763611958847,1,(6,(((4)#H7:::0.6164047855446226,3):0.02418196475935196,5):0.21941137751169684):1.1647369660654712);, with -loglik 171.92262579404783
((2)#H7:::0.5707857132716287,1,(6,(((4,#H7:::0.4292142867283712):1.8362374804380168,3):0.03396791864700508,5):0.15618778377750742):9.164596733534673);, with -loglik 191.8024060628934
((2,(4,(3)#H7:::0.552955758295583):0.08920119879051364):0.12394534351110142,1,(6,(#H7:::0.44704424170441703,5):1.6196913264642783):1.0063798767677288);, with -loglik 214.4419238447719
((2,(4,(3,(5)#H7:::0.5083511212165599):1.112351178951459):0.21002101279785773):0.048207047098199174,1,(6,#H7:::0.49164887878344005):8.633372015839782);, with -loglik 231.03858140756645
