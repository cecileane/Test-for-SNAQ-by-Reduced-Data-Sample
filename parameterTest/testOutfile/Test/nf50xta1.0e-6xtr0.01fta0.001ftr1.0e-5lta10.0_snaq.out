((2,(4,(3,((6)#H7:::0.6922899546046088,5):1.739019537243421):0.45989628065524363):0.6610706797906918):8.879032997766629,1,#H7:::0.3077100453953911); -Ploglik = 128.22642796014017
 Dendroscope: ((2,(4,(3,((6)#H7,5):1.739019537243421):0.45989628065524363):0.6610706797906918):8.879032997766629,1,#H7);
 Elapsed time: 11.381312101 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 ((2,(4,(3,((6)#H7:::0.6922899546046088,5):1.739019537243421):0.45989628065524363):0.6610706797906918):8.879032997766629,1,#H7:::0.3077100453953911);, with -loglik 128.22642796014017
 (2,((3,(5,(6)#H7:::0.6922885428890575):1.739042770218895):0.4598966544335186,4):0.6610694049213363,(1,#H7:::0.30771145711094255):8.878969856269661);, with -loglik 128.22642796573987
-------
